src/main/scripts/src/app/core/rest/user.service.ts

```diff
@@ -19,18 +19,13 @@
  */
 
 import {Injectable} from '@angular/core';
-import {HttpClient} from '@angular/common/http';
 
-@Injectable()
-export class RolesService {
-
-  constructor(private http: HttpClient) {
-  }
+import {Observable} from 'rxjs/Observable';
 
-  // TODO communicate with SecurityService through REST API
-
-  private static apiPrefix(organizationCode: string): string {
-    return `/${API_URL}/rest/roles/organizations/${organizationCode}/`;
-  }
+@Injectable()
+export class UserService {
 
+    public getUsers(): Observable<string[]> {
+        return Observable.of(['alicak', 'kubedo', 'jkotrady', 'kulexpipiens']);
```

Comment:
This method should return a list of user objects. `User` interface (similar to the [back-end one](https://github.com/Lumeer/engine/blob/devel/api/src/main/java/io/lumeer/api/model/User.java)) needs to be created in `dto` folder:
```typescript
export interface User {
  id: string;
  username: string;
  groups: string[];
}
```